class/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, specialization, age = 18, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] Kindly remove this `id`.

- [ ] The `parent_permission` should be assigned correctly in the `#super` method.

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
  ```